### PR TITLE
Better articulation of the relationship between Office JS APIs and Office Scripts APIs

### PR DESCRIPTION
**Pull request by AlexJerabek**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[b08fbe7](https://github.com/OfficeDev/office-scripts-docs/commits/b08fbe7d66adb1c99d6864df5ae74413a0b9e9b6)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/resources/add-ins-differences.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-AddInsDiffs/docs/resources/add-ins-differences.md) | :warning:Warning | | [Details](#user-content-5638ec4f-b93b6169)

<a id="5638ec4f-b93b6169"></a>
### [docs/resources/add-ins-differences.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-AddInsDiffs/docs/resources/add-ins-differences.md)
 - **Line 28**: **[Warning-[InvalidFileLink](https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-links?branch=master)]** `````Invalid file link:(~/excel-async-model.md).`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C6%5C1%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202006011713550134-117%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=2rNv%2BEpSBwDyIk1AvNDR6fvL4yiE8whJB%2F5zRWQdZ1E%3D&st=2020-06-01T17%3A10%3A07Z&se=2020-07-02T17%3A15%3A07Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/918d1c35-f3a4-631b-1edf-a1a2af22eee5?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[a6732e1](https://github.com/OfficeDev/office-scripts-docs/commits/a6732e146a3d6e9a33d78a4c8f8a5d261ab63478)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/resources/add-ins-differences.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-AddInsDiffs/docs/resources/add-ins-differences.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C6%5C1%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202006011716487307-117%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=hHePwaJ6x4X43m657pGx5PDrx2NZbiUPE735Al7eyNg%3D&st=2020-06-01T17%3A12%3A58Z&se=2020-07-02T17%3A17%3A58Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/918d1c35-f3a4-631b-1edf-a1a2af22eee5?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/resources/add-ins-differences.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 description: 'The behavior and API differences between Office Scripts and Office Add-ins.'
-ms.date: 04/24/2020
+ms.date: 06/01/2020
 localization_priority: Normal
 ---
 
 # Differences between Office Scripts and Office Add-ins
 
-Office Add-ins and Office Scripts have a lot in common. They both offer automated control of an Excel workbook through the `Excel` namespace of the Office JavaScript API. However, Office Scripts are more limited in their scope.
+Office Add-ins and Office Scripts have a lot in common. They both offer automated control of an Excel workbook a JavaScript API. However, Office Scripts use a specialized, synchronous version of the Office JavaScript API.
```
Comment (Rick-Kirkham) [inline docs/resources/add-ins-differences.md:10]: Maybe this is cleared up elsewhere in the article, but I was confused by the fact that this sentence seems to say that the Office Scripts APIs are synchronous but line 28 below refers to "Office Scripts Async APIs".

**Comment by o365devx** [comment]
Docs Build status updates of commit _[a511541](https://github.com/OfficeDev/office-scripts-docs/commits/a51154136f310ab66af0c260a60b4aa6c3cbf861)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/resources/add-ins-differences.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-AddInsDiffs/docs/resources/add-ins-differences.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C6%5C1%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202006011741594927-117%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=4fDvYiTmMpS7Ks4RZ9RcOiUmPhAuRM3WQ8wu0OltlwM%3D&st=2020-06-01T17%3A38%3A07Z&se=2020-07-02T17%3A43%3A07Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/918d1c35-f3a4-631b-1edf-a1a2af22eee5?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/resources/add-ins-differences.md (hunk 1 above)
Comment (AlexJerabek) [reply docs/resources/add-ins-differences.md]: There's two API models: standard and async. The standard (sync) model is just "Office Scripts API", whereas the other one is "Office Scripts Async API". The async model is not recommended for most users and needs special `Excel.run` code blocks to run. However, part of the reason for including that model was to better allow add-in code to run in Office Scripts. 

I pushed a new change to try to clarify this. Please let me know if there needs to be more.

**Pull request merged**